static/src/javascripts/projects/commercial/modules/article-body-adverts.js

```diff
@@ -74,17 +66,7 @@ const addArticleAds = (count: number, rules: Object): Promise<number> => {
             .slice(0, Math.min(paras.length, count))
             .map((para: Node) => {
                 bodyAds += 1;
-                return insertAdAtPara(
-                    para,
-                    getSlotName(),
-                    getSlotType(),
-                    `inline${isOffsetingAds && bodyAds > 1 ? ' offset-right' : ''}`,
```

Comment:
I believe it is correct to remove these two optional parameters as after this change they would evaluate to `''` and `null` respectively.